returnDetailAddendumB_test.go

```diff
@@ -115,6 +116,65 @@ func testRDAddendumBString(t testing.TB) {
 	}
 }
 
+// TestParseAddendumBJSON tests parsing a known ReturnDetailAddendumB
+func TestParseAddendumBJSON(t *testing.T) {
```

Comment:
First things first, thank you for including tests!

We're actually shifting our apps to use testify for assertions. Additionally I think this would make sense as two distinct test cases. Maybe something like:
```go
func TestParseAddendumBJSON_withBusinessDate(t *testing.T) {
	var testBusinessDateJSON = `{ "id": "", "payorBankName": "", "auxiliaryOnUs": "", "payorBankSequenceNumber": "3713365076", "payorAccountName": "", "payorBankBusinessDate": "20210121" }`
	addB := ReturnDetailAddendumB{}

	require.NoError(t, json.Unmarshal([]byte(testBusinessDateJSON), &addB))

	assert.Equal(t, "3713365076", addB.PayorBankSequenceNumber)
	parsedTime, timeParseErr := time.Parse(time.RFC3339, "2021-01-21T00:00:00Z")
	if timeParseErr != nil {
		t.Fatalf("Unable to parse test time: %v", timeParseErr)
	}
	assert.Equal(t, parsedTime.String(), addB.PayorBankBusinessDate.String())
	marshalledDate, marshalDateErr := json.Marshal(addB)
	if marshalDateErr != nil {
		t.Fatalf("Unable to marshal ReturnDetailAddendumB to JSON: %v", marshalDateErr)
	}
	assert.Contains(t, string(marshalledDate), "2021-01-21T00:00:00Z")
}

func TestParseAddendumBJSON_withoutBusinessDate(t *testing.T) {
	var testNoBusinessDateJSON = `{ "id": "", "payorBankName": "", "auxiliaryOnUs": "", "payorBankSequenceNumber": "3713365088", "payorAccountName": "", "payorBankBusinessDate": "" }`
	addB := ReturnDetailAddendumB{}

	require.NoError(t, json.Unmarshal([]byte(testNoBusinessDateJSON), &addB))

	assert.Equal(t, "3713365088", addB.PayorBankSequenceNumber)
	assert.True(t, addB.PayorBankBusinessDate.IsZero())
	marshalled, marshalErr := json.Marshal(addB)
	if marshalErr != nil {
		t.Fatalf("Unable to marshal ReturnDetailAddendumB to JSON: %v", marshalErr)
	}
	assert.NotContains(t, string(marshalled), "0001-01-01")
}
```